devel/libfilezilla/Portfile

```diff
@@ -5,6 +5,7 @@ PortGroup           cxx11 1.1
 
 name                libfilezilla
 version             0.11.0
+revision            1
```

Comment:
I guess bumping revision is not necessary as it's a build fix?

Reply:
@yan12125  Not very familiar with how it works with `Portfile` policy. Should I bump back the revision?  

Moreover how do I match the Commit Messages policy now that I already pushed my commit? @raimue

Reply:
I'm new here too :-) Here's a previous feedback I got: https://github.com/macports/macports-ports/pull/771#issuecomment-329507471
In my memory, ```std::foobar()``` and ```foobar()``` functions are identical, so this sounds like a case that rebuilding isn't necessary.
Removing the new ```revision``` line should work. That's what I did in #771.

> Moreover how do I match the Commit Messages policy now that I already pushed my commit?

I guess I can answer it - ```git commit --amend``` and ```git push --force```